feature/bgp/policybase/otg_tests/actions_med_localpref_prepend_flow_control/actions_MED_LocPref_prepend_flow_control_test.go

```diff
@@ -227,12 +228,11 @@ func configureASLocalPrefMEDPolicy(t *testing.T, dut *ondatra.DUTDevice, policyT
 		actions.GetOrCreateBgpActions().SetLocalPref = ygot.Uint32(uint32(metric))
 		actions.PolicyResult = oc.RoutingPolicy_PolicyResultType_ACCEPT_ROUTE
 	case setMEDPolicy:
-		if strings.Contains(policyValue, "+") {
-			actions.GetOrCreateBgpActions().SetMed = oc.UnionString(policyValue)
-		} else {
-			metric, _ := strconv.Atoi(policyValue)
-			actions.GetOrCreateBgpActions().SetMed = oc.UnionUint32(uint32(metric))
-		}
+		metric, _ := strconv.Atoi(policyValue)
```

Comment:
Instead of making it default please include it as part of deviation to check for string +